bundle-workflow/src/test_workflow/put_file_into_S3.py

```diff
@@ -0,0 +1,361 @@
+# !/usr/bin/env python3
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+import argparse
+import json
+import os
+
+from src.read_write_s3 import S3Bucket
+
+
+class ConvertWrite:
```

Comment:
I am not sure what we're trying to do here, but parsing HTML output to gather test results sounds like not a great idea. Test output must be able to produce structured output that we can examine (json/xml) or a log would suffice.

Reply:
💯   A human readable report while laudable is an incredibly complex requirement, and I'm not sure that it serve us to include one at this time.  The json response from mensor while domain specific is not so unreadable that it cannot be used.  Reporting performance data that is broadly useful is a complex task that is out of scope for near term deliverable.

No matter what decision you make, we should include the mensor json output in the artifacts.